src/Admins/SearchAdmin.php

```diff
@@ -12,13 +16,34 @@
 class SearchAdmin extends ModelAdmin
 {
     private static $managed_models = [
-        SearchClass::class
+        SearchClass::class,
+        Elevation::class,
+        ElevatedItem::class,
     ];
 
     private static $menu_icon_class = 'font-icon-search';
 
     private static $url_segment = 'searchadmin';
 
     private static $menu_title = 'Search';
+
+    public function getEditForm($id = null, $fields = null)
+    {
+        $oldImportFrom = $this->showImportForm;
+        $this->showImportForm = false;
+        /** @var GridField $gridField */
+        $form = parent::getEditForm($id, $fields);
+        $this->showImportForm = $oldImportFrom;
+
+        if ($this->modelClass === ElevatedItem::class) {
+            $gridField = $form->Fields()->dataFieldByName($this->sanitiseClassName($this->modelClass));
+
+            $gridField
+                ->getConfig()
+                ->addComponent(new GridFieldOrderableRows('Rank'));
```

Comment:
Maybe also add inline editing if it's possible? Saves users from clicking "new" etc.